packages/desktop/views/dashboard/send-flow/send-flow-route.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum SendFlowRoute {
+    SelectToken = 'selectToken',
+    InputTokenAmount = 'inputTokenAmount',
+    InputDestination = 'inputDestination',
+    TransactionSummary = 'transactionSummary',
```

Comment:
Admittedly the technical spec. isn't where it should be, but these names should be consistent with what is outlined by the tasks, i.e. `SelectAsset`, `SelectNetworkAndRecipient`, `SelectAmount` (alt. `SetAmount`), `SendDetails`. I do think `TransactionSummary` is nice though.

Regardless, whatever we go with here must be reflected in the technical spec and the tasks that we create that use component names.